Bottom: de795530562064436462399dfe158865988794ad
Top:    a68a4d08c4aa027858cc023f476306031723446b
Author: Ian Kent <ikent@redhat.com>
Date:   2013-09-12 15:26:21 +0800

Refresh of autofs-5.0.7-teach-dumpmaps-to-output-simple-key-value-pairs.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 16ea31a..07abd68 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1725,7 +1725,7 @@ static void usage(void)
 		"	-f --foreground do not fork into background\n"
 		"	-r --random-multimount-selection\n"
 		"			use ramdom replicated server selection\n"
-		"	-m --dumpmaps [<map type> <map name]\n"
+		"	-m --dumpmaps [<map type> <map name>]\n"
 		"			dump automounter maps and exit\n"
 		"	-n --negative-timeout n\n"
 		"			set the timeout for failed key lookups.\n"
